Gemfile

```diff
@@ -51,6 +51,7 @@ gem 'mixlib-authentication'
 gem 'pundit'
 
 gem 'redcarpet' # markdown parsing
```

Comment:
Do you still need `redcarpet` if you're now using `commonmarker` instead?
Just making sure you didn't forget to get rid of a potentially useless dependency ;)

Reply:
Unfortunately yes.  There are other methods depending on Redcarpet other than just the rendering of markdown.